docs/2.build/2.smart-contracts/quickstart.md

```diff
@@ -435,6 +435,6 @@ At the time of this writing, this example works with the following versions:
 - node: `20.18.0`
 - rustc: `1.81.0`
 - near-cli-rs: `0.15.1`
-- cargo-near: `0.10.1`
+- cargo-near: `0.13.2`
```

Comment:
changes in https://github.com/near/docs/pull/2390/files were somewhat inconsistent, by suggesting to use option, but not changing cmd version